content/desktop/contributing-and-collaborating-using-github-desktop/working-with-your-remote-repository-on-github-or-github-enterprise/configuring-notifications-in-github-desktop.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Configuring notifications in GitHub Desktop
+shortTitle: Configuring notifications
+intro: '{% data variables.product.prodname_desktop %} will keep you up-to-date with notifications about events that occur in your pull request branch.'
+versions:
+  fpt: '*'
+---
+## About notifications in {% data variables.product.prodname_desktop %}
+
+{% data variables.product.prodname_desktop %} will show a system notification for events that occur in the currently selected repository. Notifications will be shown when:
+
+- Pull request checks have failed.
+- A pull request review is left with a comment, approval, or requested changes.
+
+Clicking the notification will switch application focus to {% data variables.product.prodname_desktop %} and provide more detailed information.
+
+## Pull request checks failed notifications
```

Comment:
I was thinking either "Notifications about pull request check failures" or "Pull request check failure notifications", leaning toward the former, might be a bit simpler? I can foresee someone getting confused by the wording, so maybe leading with 'Notifications' would be clearer.